lib/active_model/serializer/adapter/fragment_cache.rb

```diff
@@ -41,31 +33,38 @@ def fetch
 
         private
 
-        # Given a serializer class and a hash of its cached and non-cached serializers
+        def serialize(object, serializer_class)
+          SerializableResource.new(
+            object,
+            serializer: serializer_class,
+            adapter: adapter.class
+          ).serializable_hash
+        end
+
+        # Given a hash of its cached and non-cached serializers
         # 1. Determine cached attributes from serializer class options
         # 2. Add cached attributes to cached Serializer
         # 3. Add non-cached attributes to non-cached Serializer
-        def cached_attributes(klass, serializers)
-          attributes            = serializer.class._attributes
-          cached_attributes     = (klass._cache_only) ? klass._cache_only : attributes.reject { |attr| klass._cache_except.include?(attr) }
+        def cached_attributes(serializers)
```

Comment:
looks like this method should really be called `cache_attributes` as it is neither idempotent nor does it is a Query; rather it is a Command